src/project/internal/notationproject.h

```diff
@@ -55,11 +56,21 @@ class NotationProject : public INotationProject
     NotationProject();
     ~NotationProject() override;
 
-    io::path path() const override;
-
     Ret load(const io::path& path, const io::path& stylePath = io::path(), bool forceMode = false) override;
     Ret createNew(const ProjectCreateOptions& projectInfo) override;
 
+    io::path path() const override;
```

Comment:
It is a bit confusing that this method is not part of IFileInfoProvider

Reply:
I did not put it in IFileInfoProvider because that would require to use io::path in the engraving module, and I thought we have avoided that so far (and I think IFileInfoProvider should be in the engraving module because its purpose is to provide the MasterScore and EngravingProject with all necessary info). 
But if we think it's okay to use io::path in engraving, then I could add `path()` to the IFileInfoProvider interface, although I wonder whether it will ever be used from there.

Reply:
You can freely use the framework in the engraving module :) The main restriction is not to use Qt in the engraving module